src/react/containers/EditorContainer.js

```diff
@@ -74,6 +95,29 @@ class EditorContainer extends Component {
     this.setState({ editorState: newEditorState });
   }
 
+  onSelectUsersChange(value) {
+    this.setState({
+      selectedUsers: value,
+    });
+  }
+
+  onSelectAuthorChange(value) {
+    this.setState({
+      selectedAuthor: value,
+    });
+  }
+
+  getUsers(text, callback) {
+    const { config } = this.props;
+    getAuthors(text, config)
+      .timeout(10000)
```

Comment:
Is this 10,000 seconds? Seems a bit much.

Reply:
Milliseconds, so its 10 seconds :)

Reply:
*wipes brow*